### PR TITLE
Create git.md

### PR DESCRIPTION
**Pull request by leon-lopes**

### CODE REVIEW

File: config_management/git.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+### Code Versioning
+Version control is simply the control of changes to project files, in order to maintain a history of everything that has been created and changed since the code was created.
+
+### Why?
+Version control can avoid several problems, for example, files or documents are altered by two people in parallel and one overwrites the previous change, it also prevents the loss of functional files that have been altered and presented problems etc.
+
+### Benefits
+Having an efficient version control still helps to maintain the quality of the product, helping to keep all customers with the same version of the product, or if different versions of the same product are needed if any of the customers has special needs.
```
Comment (jasondavindev) [inline config_management/git.md]: developer instead of customer

File: config_management/git.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+For project management, in addition to allowing several people to work in parallel on a project, it also allows you to work on a project in different versions, for example, part of the team finalizing version 1.0 while another part is already working on version 2.0.
```
Comment (jasondavindev) [inline config_management/git.md]: without `you` for impersonal sentences

File: config_management/git.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+* History control
+* Team work
+* Version marking
+* Recover of versions
+* Project branching
+
+---
+
+### GITHUB
+GitHub is a website that opens development version control software through the Git system.
```
Comment (jasondavindev) [inline config_management/git.md]: Is a tool instead of website

File: config_management/git.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+When you click on it, GitHub itself automatically analyzes your branch's commit with the most recent commit from the master, to check for conflicts (different information on the same line of code).
+
+After that add two people to review the code and then just complete the pull request.
+
+---
+
+## Code Review Process
+Code review is a code review practice that is widely used and important that can bring many benefits to the product and the team developing it.
+When you are chosen to review the code, follow the steps:
```
Comment (jasondavindev) [inline config_management/git.md:70]: Impersonal

File: config_management/git.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+### GITHUB
+GitHub is a website that opens development version control software through the Git system.
+Through this system, two or more people can edit existing files or create without risking that their changes will be erased by the modifications of other team members. This is useful for the most different types of projects.
+
+### Commits
+Commits are points within a project's editing history and are made up of a set of file changes. They include a description that details exactly what has been changed in the code and a point to restore a particular version of the project.
+
+### Branches
+Branches are for programmers to be able to work on larger functionalities, which demand more development time, without affecting the current version of the application. GitHub creates a copy of the directory which can then be developed in isolation, without affecting the central repository or other branches.
+When work is completed on a branch, it can be joined to other branches through the process known as merge. In case you want to join this isolated branch with the central repository.
```
Comment (jasondavindev) [inline config_management/git.md:29]: Impersonal

**Comment by jasondavindev** [comment]
It's missing a summary to explain the overview of gitflow: the developer opens a branch to develop their task, writes the code, commits, open the pull request, requets the code reviewers and finally merge them.

File: config_management/git.md (hunk 2 above)
Comment (M-Fernandes) [reply config_management/git.md]:
```suggestion
For project management, in addition to allowing several people to work in parallel on a project, **it also allows working with different versions of a project**, for example, part of the team finalizing version 1.0 while another part is already working on version 2.0.
```

**Review by M-Fernandes — APPROVED** [review]

File: config_management/git.md (hunk 1 above)
Comment (leon-lopes) [reply config_management/git.md]: Done

File: config_management/git.md (hunk 2 above)
Comment (leon-lopes) [reply config_management/git.md]: I fixed it for a good part, some I kept

File: config_management/git.md (hunk 3 above)
Comment (leon-lopes) [reply config_management/git.md]: Done

File: config_management/git.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+When work is completed on a branch, it can be joined to other branches through the process known as merge.
+
+### Pull Request
+Pull Request, the person responsible for managing the project will receive a request to analyze the request and decide whether or not to accept a certain contribution. This serves both to merge branches with the central repository and to allow someone to copy the files to their personal computer.
+The method is also useful for initiating code reviews and general discussions about certain changes before implementing them in the central repository.
+
+---
+
+## Commit Process
+When added, edited or deleted a file, you are committing and adding it to your branch. This process of adding commits keeps track of your progress while you work on a branch.
```
Comment (jasondavindev) [inline config_management/git.md]:
```suggestion
When added, edited or deleted a file, commits are added to the branch. This process of adding commits keeps track of the progress while working on a branch.
```

**Review by jasondavindev — APPROVED** [review]

**Pull request merged**